### PR TITLE
[Spike | `use-ethBridge`] Add `erc721` examples

### PR DESCRIPTION
**Pull request by aidan-starke**
## Summary

 - Add `bridgeTRNToETH` example
 - Add `bridgeETHToTRN` example

## Checklist

- [x] Add description
- [ ] Tag related issue(s)

### CODE REVIEW

**Comment by ken-centrality** [comment]
@KarishmaBothara please review this

**Comment by KarishmaBothara** [comment]
> @KarishmaBothara please review this

Trying to mint some ERC721 tokens for tnl/seeker contract using Aidan's repo - https://github.com/aidan-starke/fv-mock-nfts

Have used Alice's key in .env file... however 

```
 npx hardhat mock_mint --network goerli --contract "tnl" --address "0x25451A4de12dcCc2D166922fA938E900fCc4ED24" --amount 1
```



> @KarishmaBothara please review this

so I minted one token for bob using Aidan's repo - https://github.com/aidan-starke/fv-mock-nfts

src/erc20/bridgeETHToTRN --asset=3944

Screenshot 2023-07-21 at 1 01 44 PM


But when I try to run the bridging example with bob's private key

 CALLER_PRIVATE_KEY=0x7..cf  pnpm call src/erc20/bridgeETHToTRN --asset=3944

I end up with error
Screenshot 2023-07-21 at 1 03 39 PM

**Comment by aidan-starke** [comment]
@KarishmaBothara can you please try again, the command should be like
```
pnpm call src/erc721/bridgeETHToTRN --tokenId=...
```

File: examples/substrate/use-ethBridge/src/erc20/bridgeETHToTRN.ts [hunk 1]
```diff
@@ -31,10 +33,19 @@ async function main() {
 	const syloContract = getERC20Contract(SyloAsset.address, wallet);
 	const { bridgeContract, erc20PegContract } = getBridgeContracts("goerli", wallet);
 
+	// Bridge requires a small fee
+	let sendMessageFeeBN: BigNumber | undefined;
+	// This can fail when using `defaultProvider`
+	try {
+		sendMessageFeeBN = await bridgeContract.sendMessageFee();
+	} catch (error: any) {
+		if (error?.code === "CALL_EXCEPTION") await main();
```
Comment (vldolot) [inline examples/substrate/use-ethBridge/src/erc20/bridgeETHToTRN.ts:42]: Perhaps, can add an explanation on this line.

File: examples/substrate/use-ethBridge/src/erc20/bridgeTRNToETH.ts [hunk 2]
```diff
@@ -24,6 +25,16 @@ async function main() {
 	const wallet = new Wallet(env.CALLER_PRIVATE_KEY, provider);
 	const { bridgeContract } = getBridgeContracts("goerli", wallet);
 
+	// Bridge requires a small fee
+	let bridgeFee: BigNumber | undefined;
+	// This can fail when using `defaultProvider`
+	try {
+		bridgeFee = await bridgeContract.bridgeFee();
+	} catch (error: any) {
+		if (error?.code === "CALL_EXCEPTION") await main();
```
Comment (vldolot) [inline examples/substrate/use-ethBridge/src/erc20/bridgeTRNToETH.ts:34]: Same here

**Review by vldolot — COMMENTED** [review]
LGTM, just needed some comments to make code more readable.

**Comment by vldolot** [comment]
> @KarishmaBothara can you please try again

I'm gonna try this one as well.. it might just be a similar issue that I encountered using `pnpm`

**Review by KarishmaBothara — APPROVED** [review]

**Comment by KarishmaBothara** [comment]
[0x3d4652430d0589d73fe03e3ac230e8e6ed25f8e49fb049bbd94193fd17c12e78](https://goerli.etherscan.io/tx/0x3d4652430d0589d73fe03e3ac230e8e6ed25f8e49fb049bbd94193fd17c12e78)

**Pull request merged**